_articles/how-to-contribute.md

```diff
@@ -505,7 +505,7 @@ If you haven't gotten a response in over a week, it's fair to politely respond i
 
 **Don't** reach out to that person privately; remember that public communication is vital to open source projects.
 
-If you make a polite bump and still nobody responds, it's possible that nobody will respond, ever. It's not a great feeling, but don't let that discourage you. It's happened to everyone! There are many possible reasons why you didn't get a response, including personal circumstances that may be out of your control. Try to find another project or way to contribute. If anything, this is a good reason not to invest too much time in making a contribution before other community members are engaged and responsive.
+If you make a polite bump and still nobody responds, it's possible that nobody will respond, ever. It's not a great feeling, but don't let that discourage you. It happened to everyone! There are many possible reasons why you didn't get a response, including personal circumstances that may be out of your control. Try to find another project or way to contribute. If anything, this is a good reason not to invest too much time in making a contribution before other community members are engaged and responsive.
```

Comment:
I don't think this is incorrect, actually. 

> It's happened to everyone

is a contraction of "It has happened to everyone" which sounds better than "it happened to everyone", to me. Thoughts?

Reply:
Uuuhm, didn't know this 😅.
Should close the PR maybe.

Reply:
no worries, thanks for understanding!